dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3537,6 +3526,22 @@ else if (oldFile.exists()) {
 
         } // end syncronized block
 
+        if(contentlet.isFileAsset()){
+          
+          FileAsset asset = APILocator.getFileAssetAPI().fromContentlet(contentlet);
+          
+          Logger.info(this.getClass(), "file Asset saved:" + contentlet.getIdentifier());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "file Asset saved:" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)